admin/tests/integration/components/certification/certification-details-competence-test.js

```diff
@@ -31,12 +31,10 @@ module('Integration | Component | certification-details-competence', function(ho
   test('it renders', async function(assert) {
     // given
     this.set('competenceData', competence('ok', 'ko', 'partially'));
-    this.set('externalAction', () => {
-      return resolve();
-    });
+    this.set('externalAction', () => resolve());
 
     // when
-    await render(hbs`{{certification-details-competence competence=competenceData rate=60 juryRate=false onUpdateRate=(action externalAction)}}`);
+    await render(hbs`<Certification::CertificationDetailsCompetence @competence={{competenceData}} rate=60 @juryRate=false @onUpdateRate=(action externalAction)/>`);
```

Comment:
**Question:** Les {{}} sont elles optionnelles? On les a mises sur d'autres args

```suggestion
    await render(hbs`<Certification::CertificationDetailsCompetence @competence={{competenceData}} rate=60 @juryRate={{false}} @onUpdateRate=(action externalAction)/>`);
```

Reply:
Je n'arrive pas à être catégorique sur leur utilisation obligatoire ou pas. Je crois que non, quand in utilise `hbr`.

Reply:
Finalement, je crois que je tiens [ma réponse](https://guides.emberjs.com/release/components/conditional-content/#toc_usage-1) :
![image](https://user-images.githubusercontent.com/265963/99319669-3c679d80-286a-11eb-913e-fab5758f16bb.png)